cEP-0020.md

```diff
@@ -0,0 +1,462 @@
+## Background
+
+### Newcomer Experience
+
+**The problems we have:**
+
+- How to attract more newcomers?
+- How to cope with the increasing amounts of newcomers?
+- How to teach and help newcomers to learn faster so
+  we can keep working on coala at a high pace?
+- How to improve newcomer -> dev/maintainer ratio?
+- How to increase amount of feedback we get?
+- How to use those feedback to improve coala and its processes?
+
+## Why Gamification?
+
+As coala tries to be a welcoming organization for newcomers, it gives them
+clear pathways and as much direct assistance as possible. However, it is
+already known that newcomers face many barriers while attempting to contribute
+to open source for the first time, and additional barriers while attempting
+to contribute to projects run by the coala organization.
+
+Some barriers they face include orientation issues that can potentially
+demotivate newcomers from placing their first contribution. On the other
+hand, gamification is widely used to engage and motivate people to
+accomplish tasks and improve their performance.
+
+Therefore, the goal of this project is to use gamification to orient and
+motivate newcomers to overcome onboarding barriers to contribute and engage
+with coala and its community.
+
+## Gamification Introduction
+
+There will be four game design elements to help newcomers: Activities, Points,
+Badges and Levels. For each of these elements, we will have a set of rules to
+describe the operation of the gamified environment. i.e.:
+
+### Activities
+
+Let's start by defining some activities which could be tracked:
+
+- Create a bug report
+- Create newcomer difficulty issue
+- Create a bear proposal issue
+- Create a documentation issue
+- Create low difficulty issue
+- Create medium difficulty issue
+- Create pull request that fixes a bug
+- Create pull request for bear proposal
+- Create pull request for a documentation issue
+- Create pull request for newcomer issue
+- Create pull request for low difficulty issue
+- Create pull request for medium difficulty issue
+- Gitmate errors on pull request
+- CI status of the pull request
+- Review comments on pull request
+- Number of questions asked in `coala/coala` room
+- Number of answers given in `coala/coala` room
+- Number of questions asked in `coala-bears` room
+- Number of  answers given in `coala-bears` room
+- Number of questions asked in `coala/corobo` room
+- Number of answers given in `coala/corobo` room
+- Number of questions asked in `coala/community` room
+- Number of answers given in `coala/community` room
+- Number of messages in coala other rooms
+- Make use of `corobo` on Gitter
+- Introduce `.coafile` in user own repository
+- Introduce `.coafile` in an organisation with 50 or more stars for its repositories
+- Introduce CI in the users own repository to run coala using Travis CI
+- Setup the CIs used by coala to your fork
+
+### Points
+
+Points are a simple tool to reward behavior, and they provide an excellent
```

Comment:
"reward behavior" sounds a bit weird here, kinda as if you're parenting the newcomers. Maybe something more like "reward positive contributions to coala"?.